OtripleS.Web.Api.Tests.Unit/Services/AttendanceServiceTests/AttendanceServiceTests.Validation.Delete.cs

```diff
@@ -0,0 +1,97 @@
+﻿// ---------------------------------------------------------------
```

Comment:
Classname should be "AttendanceServiceTests.Validations.Delete.cs" or?

Reply:
@tobidub, @hassanhabib  thanks. Changed that.